libs/blocks/accordion/accordion.js

```diff
@@ -38,7 +39,11 @@ function createItem(accordion, id, heading, num) {
   const para = panel.querySelector('p');
   const text = para ? para.textContent : panel.textContent;
 
-  const dt = createTag('dt', { role: 'heading', 'aria-level': 3 }, button);
+  const dt = createTag(
+    'dt',
+    { role: 'heading', 'aria-level': 3 },
+    hTag ? createTag(hTag.tagName, { class: 'accordion-heading' }, button) : button,
```

Comment:
The nested createTag makes this a little difficult to read.
Should the `aria-level` change to match the heading level?

Reply:
> Should the aria-level change to match the heading level?

@Brandon32 now that you mention it, I'm wondering if the aria level/role should even be set at all if an h tag is present, since those attributes are there to tell screen readers to interpret elements as h tags 🤔